### PR TITLE
Update russian translation

### PR DESCRIPTION
**Pull request by Blueberryy**
- Fixed some little typos.
- Added untranslated strings.

### CODE REVIEW

**Comment by Blueberryy** [comment]
I don't know what kind of review you will do but okay

File: locale/shine/extensions/votealltalk/ruRU.json [hunk 1]
```diff
@@ -1,3 +1,14 @@
...
+	"PLAYER_VOTED_DISABLE": "{PlayerName} выбрал отключение межкомандного общения. Необходимо ещё {VotesNeeded}.",
+	"PLAYER_VOTED_PRIVATE_ENABLE": "Вы проголосовали за включение межкомандного общения. Необходимо ещё {VotesNeeded}.",
+	"PLAYER_VOTED_PRIVATE_DISABLE": "Вы проголосовали за отключение межкомандного общения. Необходимо ещё {VotesNeeded}.",
+	"ERROR_ALREADY_VOTED_ENABLE": "Вы уже проголосовали за включение межкомандного общения.",
+	"ERROR_ALREADY_VOTED_DISABLE": "Вы уже проголосовали за отключение межкомандного общения.",
+	"ERROR_MUST_WAIT_ENABLE": "Вы должны подождать {SecondsToWait:Duration} сек. перед голосованием за включение межкомандного общения.",
+	"ERROR_MUST_WAIT_DISABLE": "Вы должны подождать {SecondsToWait:Duration} сек. перед голосованием за отключение межкомандного общения.",
+	"ERROR_CANNOT_VOTE_IN_SPECTATE": "Наблюдателям не разрешено голосовать за включение межкомандного общения.",
+	"DISABLE_ALLTALK": "Отключить межкомандное общение",
+	"ENABLE_ALLTALK": "Включить межкомандное общение"
```
Comment (Person8880) [inline locale/shine/extensions/votealltalk/ruRU.json:13]: These last 2 strings are used on the vote menu button, and end up quite a bit larger than the button is. Is there a shorter way of writing this? It's not a huge problem if not, just means the text overflows a bit.

File: locale/shine/core/ruRU.json [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
 	"NAME": "Имя",
 	"TEAM": "Команда",
 	"ERROR": "Ошибка",
 	"OK": "OK",
 	"ADMIN_MENU_SELECT_SINGLE_PLAYER": "Выберите одного игрока.",
-	"ADMIN_MENU_OPEN_WIKI": "Открыть Wiki",
+	"ADMIN_MENU_OPEN_WIKI": "Открыть вики",
 	"ADMIN_MENU": "Администрирование",
-	"Shuffle": "Shuffle",
+	"Shuffle": "Распредел.",
```
[the remark this answers is not in the document]
Comment (Blueberryy) [reply locale/shine/core/ruRU.json:12]: You can delete **.** if it broke string.

File: locale/shine/core/ruRU.json [hunk 3]
```diff
@@ -33,11 +33,11 @@
...
-	"SECOND": "{Value:Pluralise:секунда(ы)|секунд}",
-	"MINUTE": "{Value:Pluralise:минута(ы)|минут}",
-	"HOUR": "{Value:Pluralise:час(а)|часов}",
-	"DAY": "{Value:Pluralise:день(дня)|дней}",
-	"WEEK": "{Value:Pluralise:неделя(и)|недель}",
+	"SECOND": "{Value:Pluralise:сек.|сек.}",
+	"MINUTE": "{Value:Pluralise:мин.|мин.}",
+	"HOUR": "{Value:Pluralise:ч.|ч.}",
+	"DAY": "{Value:Pluralise:день(-я)|дней}",
+	"WEEK": "{Value:Pluralise:неделя(-и)|недель}",
```
[the remark this answers is not in the document]
Comment (Blueberryy) [reply locale/shine/core/ruRU.json:40]: Yeah it will just look better and short

File: locale/shine/extensions/votealltalk/ruRU.json (hunk 1 above)
Comment (Blueberryy) [reply locale/shine/extensions/votealltalk/ruRU.json:13]: Will fix asap

File: locale/shine/core/ruRU.json [hunk 4]
```diff
@@ -33,11 +33,11 @@
...
 	"TIME_SENTENCE": "{Before} и {After}",
 	"TIME_SEPARATOR": ", ",
-	"SECOND": "{Value:Pluralise:секунда(ы)|секунд}",
-	"MINUTE": "{Value:Pluralise:минута(ы)|минут}",
-	"HOUR": "{Value:Pluralise:час(а)|часов}",
-	"DAY": "{Value:Pluralise:день(дня)|дней}",
-	"WEEK": "{Value:Pluralise:неделя(и)|недель}",
+	"SECOND": "{Value:Pluralise:сек.|сек.}",
+	"MINUTE": "{Value:Pluralise:мин.|мин.}",
+	"HOUR": "{Value:Pluralise:ч.|ч.}",
```
[the remark this answers is not in the document]
Comment (Blueberryy) [reply locale/shine/core/ruRU.json:38]: because short yeah

**Comment by Person8880** [comment]
All looks good to me now, thanks again for contributing.

**Pull request merged**